assimilation_code/programs/obs_utils/obs_info.f90

```diff
@@ -222,11 +228,24 @@ program obs_info
                                              trim(get_name_for_type_of_obs(i))//", ", &
                                              oinfo(i)%count, ", ", trim(mid_string)
       else
-         write(ounit, '(A,I8)') get_name_for_type_of_obs(i), oinfo(i)%count
+         write(ounit, '(A32,I8)') get_name_for_type_of_obs(i), oinfo(i)%count
          call print_date(oinfo(i)%first_time, '.  First obs:', ounit)
          call print_date(oinfo(i)%last_time,  '.   Last obs:', ounit)
       endif
    enddo ALLTYPES
+   if (identity_obs%count > 0) then
+      if (counts_only) then
+         call compute_times(identity_obs%first_time, identity_obs%last_time, avg_string=mid_string)
+         write(ounit, '(A,I8,A,A36,I8,2A)') "'"//trim(filename_in(fnum))//"', ", &
+                                             -1, ", ", &
+                                             "IDENTITY_OBSERVATIONS"//", ", &
+                                             identity_obs%count, ", ", trim(mid_string)
+      else
+         write(ounit, '(A32,I8)') "IDENTITY_OBSERVATIONS", identity_obs%count
+         call print_date(identity_obs%first_time, '.  First obs:', ounit)
+         call print_date(identity_obs%last_time,  '.   Last obs:', ounit)
+      endif
+   endif
```

Comment:
This is the block that reports on the summary of the identity observations.

Reply:
agreed.  fixed in my repo and will be pushed soon.